### DISCUSSION TITLE
Is MonkeyType named after the Infinite Monkey Theorem or is it a conicidence?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [monkeytypegame/monkeytype#5504] [Questions]
While looking stuff up about MonkeyType, I found the wikipedia page on the [Infinite Monkey Theorem](https://en.wikipedia.org/wiki/Infinite_monkey_theorem):

> The infinite monkey theorem states that a [monkey](https://en.wikipedia.org/wiki/Monkey) hitting keys at [random](https://en.wikipedia.org/wiki/Randomness) on a [typewriter](https://en.wikipedia.org/wiki/Typewriter) keyboard for an [infinite](https://en.wikipedia.org/wiki/Infinity) amount of time will [almost surely](https://en.wikipedia.org/wiki/Almost_surely) type any given text, including the complete works of [William Shakespeare](https://en.wikipedia.org/wiki/William_Shakespeare). In fact, the monkey would almost surely type every possible finite text an infinite number of times. The theorem can be generalized to state that any sequence of events that has a non-zero [probability](https://en.wikipedia.org/wiki/Probability) of happening will almost certainly occur an infinite number of times, given an infinite amount of [time](https://en.wikipedia.org/wiki/Time) or a [universe](https://en.wikipedia.org/wiki/Universe) that is [infinite in size](https://en.wikipedia.org/wiki/Shape_of_the_universe#Global_universe_structure).

**Comment by Miodec** [comment]
Indirectly. The name comes from a suggestion of a reddit user (https://www.reddit.com/r/MechanicalKeyboards/comments/gc6wx3/experimenting_with_a_completely_new_type_of/fp9lzj4/) which i believe is based on a simpsons episode (as hes referencing "it was the best of times, it was the blurst of times"), which is referencing the infinite monkey theorem.

I just liked the name, didn't look into the infinite monkey theory back then.
